dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ExecutorServiceImpl.java

```diff
@@ -607,7 +607,7 @@ private Map<String, Object> insertCommand(User loginUser, Integer instanceId, lo
     }
 
     /**
-     * check if sub processes are offline before starting process definition
+     * check whether sub processes are offline before starting process definition
```

Comment:
`sub processes` or `subprocesses`? I think either one is ok but just keep it consistent.

Reply:
I think with subprocesses